website/docs/reference/resource-configs/target_schema.md

```diff
@@ -53,38 +55,10 @@ snapshots:
 
 </File>
 
-### Use a target-aware schema
-Use the [`{{ target }}` variable](/reference/dbt-jinja-functions/target) to change which schema a snapshot <Term id="table" /> is built in.
-
-Note: consider whether this use-case is right for you, as downstream `refs` will select from the `dev` version of a snapshot, which can make it hard to validate models that depend on snapshots (see above [FAQ](#faqs))
-
-<File name='dbt_project.yml'>
-
-```yml
-snapshots:
-  +target_schema: "{% if target.name == 'prod' %}snapshots{% else %}{{ target.schema }}{% endif %}"
-
-```
-
-</File>
+<VersionBlock lastVersion="1.8" >
 
 ### Use the same schema-naming behavior as models
 
-Leverage the [`generate_schema_name` macro](/docs/build/custom-schemas) to build snapshots in schemas that follow the same naming behavior as your models.
-
-Notes:
-* This macro is not available when configuring from the `dbt_project.yml` file, so must be configured in a snapshot config block.
-* Consider whether this use-case is right for you, as downstream `refs` will select from the `dev` version of a snapshot, which can make it hard to validate models that depend on snapshots (see above [FAQ](#faqs))
-
+For native support of environment-aware snapshots, upgrade to dbt Core version 1.9+ and remove any existing `target_schema` configuration. 
```

Comment:
@graciegoheen : out of curiousity,  if existing `target_schema` config is not removed/omitted , does the user run into any wonky behavior?

Reply:
Nope - it will continue to behave as it has in earlier versions - the snapshot will build to a single schema, regardless of the environment that you're running in